Source/OIDAuthorizationService.m

```diff
@@ -574,10 +574,12 @@ + (void)performTokenRequest:(OIDTokenRequest *)request
         return;
       }
 
-      // OpenID Connect Core Section 3.1.3.7. rule #3
-      // Validates that the audience of the ID Token matches the client ID.
+      // OpenID Connect Core Section 3.1.3.7. rule #3 & Section 2 azp Claim
+      // Validates that the audience of the ID Token matches the client ID unless a matching azp
```

Comment:
nit, I might phrase it as
"Validates that the aud (audience) Claim contains the client ID, or that the azp (authorized party) Claim matches the client ID."

Reply:
Yes, that's better.